lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -42,7 +42,7 @@
             - "State which should a host to be in after successful completion."
         choices: [
             'present', 'absent', 'maintenance', 'upgraded', 'started',
-            'restarted', 'stopped', 'reinstalled', 'iscsidiscover'
+            'restarted', 'stopped', 'reinstalled', 'iscsidiscover', 'iscsilogin'
```

Comment:
Please add a note that `iscsilogin` is supported since 2.4. Something like:

```yaml
state:
         description:
             - "State which should a host to be in after successful completion."
             - "I(iscsilogin) and I(iscsidiscover) are supported since version 2.4."
```